packages/client/src/components/DeleteToolConfirmation/DeleteToolConfirmation.component.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import './DeleteToolConfirmation.style.css';
+import getApiBaseUrl from '../../utils/getApiBaseURL';
+import { UserAuth } from '../../firebase/AuthContext';
+
+export const DeleteConfirmation = () => {
+  const location = useLocation();
+  const { id } = location.state;
+  const { user } = UserAuth();
+  function deleteToolByFetch() {
+    fetch(`${getApiBaseUrl()}/admin/tools/${id}`, {
+      method: 'DELETE',
```

Comment:
Do you think you can fix the API? Since you didn't include the `onDelete('CASCADE')` option when defining the relationship between the tables, you can either write a migration to fix that or you can delete from the categories_tools and favorites table manually.